notebooks/ICESat-2_Cloud_Access/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -53,7 +53,7 @@
     "### **Example of end product** \n",
```

Comment:
<p><code>earthaccess</code> API has <code>earthaccess.search_datasets</code>. If we are updating to the <code>search_data</code> method,  I think we should use <code>search_datasets</code> as well.  This will save confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I take your point Andy, one thing I am not sure about is if we make the update to `Query = earthaccess.search_datasets`, what are the equivalent commands for `Query.hits()` and `Query.fields(['ShortName', 'Version']).get(10)` in order to display the relevant metadata. @betolink do you have any suggestions?

Reply:
There are no equivalents at the top level, I think of `search_datasets()` and `search_data()` as short hand method for those not familiar with the query classes. In the documentation I'm using both but I think we should guide our users  towards the top level API unless they need a more involved query to CMR. Also, `search_data()` tell us about the hits and we limit the total results using `count=`

Reply:
I have updated the notebook to use search_datasets.